presto-tests/src/test/java/io/prestosql/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -3393,6 +3393,23 @@ public void testLogicalExplain()
         assertEquals(getOnlyElement(result.getOnlyColumnAsSet()), getExplainPlan(query, LOGICAL));
     }
 
+    @Test
+    public void testJoinDistribution()
+    {
+        /*
+            This test makes sure that the query plan includes the distribution info for joins.
+            The webui live plan displays the distribution type for joins, and the plan details
+            need to be structured like 'Distribution: <type>' for this feature to work.
+        */
```

Comment:
We lean towards `//` comments. I am not sure there is a good reason, but worth staying consistent.

```suggestion
        // This test makes sure that the query plan includes the distribution info for joins.
        // The webui live plan displays the distribution type for joins, and the plan details
        //  need to be structured like 'Distribution: <type>' for this feature to work.
```